apps/condo/cypress/objects/Auth.js

```diff
@@ -0,0 +1,195 @@
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+/* eslint-disable no-undef */
```

Comment:
We need to find way for writing tests without casting `no-undef` on each test file